src/goto-symex/symex_function.cpp

```diff
@@ -283,8 +283,9 @@ void goto_symext::symex_function_call_code(const expr2tc &expr)
   type2tc tmp_type = migrate_type(goto_function.type);
 
   if(
-    to_code_type(tmp_type).arguments.size() != arguments.size() &&
-    !to_code_type(tmp_type).ellipsis)
+    to_code_type(tmp_type).arguments.size() > arguments.size() ||
+    (0 && to_code_type(tmp_type).arguments.size() < arguments.size() &&
```

Comment:
`0` forces the second part of the guard to be false, can you remove it?

Reply:
Thanks, done. I've also reworded the error message.